cmd/tpm-v2.c

```diff
@@ -413,6 +446,8 @@ U_BOOT_CMD(tpm2, CONFIG_SYS_MAXARGS, 1, do_tpm, "Issue a TPMv2.x command",
 "    Read PCR #<pcr> to memory address <digest_addr>.\n"
 "    <pcr>: index of the PCR\n"
 "    <digest_addr>: address to store the a 32-byte SHA256 digest\n"
+"get_random <len>\n"
+"    Get random bytes.\n"
```

Comment:
Probably we should add the parameter description.

Reply:
um, isnt it self-explanatory?

Reply:
Describing parameters may be needed to upstream this commit.

Reply:
ok